docker/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "2"
+
+services:
+  redis:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    ports:
+      - 6379:6379
+    volumes:
+      - ./docker/data:/data
```

Comment:
By this, `docker/docker/data` was created.

Reply:
Thanks for the comment.
I fixed with this commit: d7b5b1cf93c41fd02308d2f3a531dfb06513e326